lib/cloudant.js

```diff
@@ -534,9 +534,11 @@ Cloudant.prototype.updateAll = function(model, where, data, options, cb) {
       return cb();
     }, function(err) {
       if (err) return cb(err);
-      mo.db.bulk({docs: docs}, function(err, result) {
-        if (result) result.count = docs.length;
-        return cb(err, result);
+      mo.db.bulk({docs: docs}, function(err) {
+        if (err) return cb(err);
+        var res = {};
```

Comment:
Nope. All other connector just returns the count of the number of instances updated, so this fix is to make cloudant consistent with the rest.